src/Inquirer/Validators/CreditCardNumberValidator.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System.Text.RegularExpressions;
+
+namespace InquirerCore.Validators
+{
+    public class CreditCardNumberValidator : IValidator
+    {
+        public bool Validate(string value)
+        {
+            //Remove spaces and dashes
+            string cleanValue = new Regex(@"[\s-]+")
+                .Replace(value, "");
+
+            // It matches a 16 digits number?
+            return new Regex(@"^\d{16}$")
```

Comment:
According to this [wiki page](https://en.wikipedia.org/wiki/Payment_card_number) to be a valid credit card number we should use the Luhn algorithm. I found this [page](https://www.freeformatter.com/credit-card-number-generator-validator.html#validate) with some good explanation.